src/libs/actions/PersistedRequests.ts

```diff
@@ -17,10 +18,61 @@ function clear() {
     return Onyx.set(ONYXKEYS.PERSISTED_REQUESTS, []);
 }
 
+/**
+ * Method to merge two arrays of OnyxUpdate elements.
+ * Elements from the old array which keys are not present in the new array are merged with the data of the new array.
```

Comment:
Sorry, I'm pretty confused by this function, and having difficulty understanding why you'd need it.